modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -69,17 +82,17 @@ const init = (lazySinks: { popup: () => Result<AlloyComponent, string>; dialog:
           shouldDisable: true
         };
       }
-      const [ key, value = '' ] = specContext.split(':');
       const contexts = editor.ui.registry.getAll().contexts;
-      const enabledInContext = Obj.get(contexts, key)
-        .fold(
-          // Fallback to 'mode:design' if key is not found
-          () => Obj.get(contexts, 'mode').map((pred) => pred('design')).getOr(false),
-          (pred) => value.charAt(0) === '!' ? !pred(value.slice(1)) : pred(value)
-        );
+      const contextResults = Arr.map(
+        Arr.filter(specContext.split(','), (spec) => spec.trim() !== ''),
```

Comment:
The AI was correct about there being a whitespace concern, but if you think about it we use this same pattern in many other places (that's likely why it was chosen for this feature). Find the common function we use for all of them and use it here, please. I think it's called `explode` or something like that.